apps/web/eslint.config.js

```diff
@@ -0,0 +1,64 @@
+import js from '@eslint/js';
```

Comment:
The `web` app config was failing on both linting and building. It seems like `typescript@5.6.2`, `eslint@8.57.1` and `typescript-eslint@8.13.0` are not compatible with each other. I didn't get to the reason, but decided to update the `web` to `eslint@9.9.0`.
I copied the ESLint config from the dashboard to the web, luckily enough, there were only ~36 errors in the web, so I decided to fix them as they were "easy".